examples/js/effects/VREffect.js

```diff
@@ -141,17 +134,35 @@ THREE.VREffect = function ( renderer, onError ) {
 				eyeWidth = eyeParamsL.renderWidth;
 				eyeHeight = eyeParamsL.renderHeight;
 
+				if ( vrDisplay.getLayers ) {
+
+					var layers = vrDisplay.getLayers();
+					if (layers.length) {
+
+						leftBounds = layers[0].leftBounds || [ 0.0, 0.0, 0.5, 1.0 ];
```

Comment:
Confirmed. The layer object returned is the one you passed in (or a copy of it.) If you pass in `null` for one of the bounds or leave it undefined you get the same thing back out. Willing to hear reasons why this should change if anyone feels strongly about it, but it seems like pretty logical behavior to me.